less/mixins.less

```diff
@@ -1 +1,37 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.buttons(@width; @bgc; @tc; @fontsize) {
+	background-color: @bgc;
+	color: @tc;
+	border-radius: 12px;
+	font-size: @fontsize;
+	text-align: center;
+    vertical-align: middle;
+    opacity: 0.6;
+    transition: 0.6s;
+	
+}
+.middle() {
+    justify-content: center;
+}
+
+.disappear () {
+    display:none;
+}
+
+.flexer() {
+    display:flex;
+}
```

Comment:
You could also add parameters to this mixin to like `flex-direction`, `flex-wrap`, `justify-content`, `align-content` and make it alot more powerful.